### PR TITLE
fix(rhosak): add logo and rename skeleton template to make it rhosak specific

### PR DESCRIPTION
**Pull request by machi1990**
~~ Opening as draft as I still need to test it using the email editor. ~~

Tested and happy with the rendering. We can go with this for now.

### CODE REVIEW

File: backend/src/main/resources/templates/Rhosak/actionRequiredBody.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
-{#include Rhosak/insightsEmailBody}
+{#include Rhosak/rhosakEmailBody}
 {#content-title}Action Required - {action.context.action_completion_time.toUtcFormat()}{/content-title}
 {#content-body}
 <tr>
     <td class="rh-content__block">
+        <div>Hello {user.firstName} {user.lastName},</div>
```
Comment (machi1990) [inline backend/src/main/resources/templates/Rhosak/actionRequiredBody.html:6]: Would love to confirm if this work, saw it as a possibility in the email template editor. It follows on my question on slack.
@gwenneg @josejulio

**Comment by machi1990** [comment]
@akoserwal @davidffrench this brings in a couple of fixes and adds the service disruption template. Looking for a review from someone in our side.

**Comment by akoserwal** [comment]
LGTM

**Comment by machi1990** [comment]
@gwenneg @josejulio would you mind having a look at this PR? Thanks

**Review by davidffrench — APPROVED** [review]
LGTM

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**